### DISCUSSION TITLE
How do you get the Icon after the Link in Menu-Entry?

### DISCUSSION THREAD

**Discussion by cnichte** [thuliteio/doks#1199] [Q&A]
How do you get the Icon after the Link in Menu.

![Bildschirmfoto 2024-03-04 um 18 27 46](https://github.com/gethyas/hyas/assets/96977775/25ea7683-99a3-42dd-80a0-2570da8cb04d)

I looked at:

https://github.com/gethyas/getdoks.org/blob/main/config/_default/menus/menus.en.toml

```
[[main]]
  name = "Hyas"
  url = "https://gethyas.com"
  post = '<svg xmlns="http://www.w3.org/2000/svg" class="ms-1 mb-1 icon icon-tabler icon-tabler-external-link" width="20" height="20" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z" fill="none"/><path d="M12 6h-6a2 2 0 0 0 -2 2v10a2 2 0 0 0 2 2h10a2 2 0 0 0 2 -2v-6" /><path d="M11 13l9 -9" /><path d="M15 4h5v5" /></svg>'
  weight = 50
```

But adding that post="..." Attribute  alone has no effect. What do I miss here? Any suggestions are welcome.

**Comment by h-enk** [comment]
Currently, you would have to override `header.html`. So copy `node_modules/@hyas/doks-core/layouts/partials/header/header.html` over to `/layouts/partials/header/header.html` and edit for the following:

https://github.com/gethyas/getdoks.org/blob/main/layouts/partials/header/header.html#L153

Better header navigation (including support for icons before and after a link) will be part of Doks 1.5.
